content/widgets.md

```diff
@@ -58,6 +58,18 @@
 * Update the preferred content size
 * Wait for `viewWillTransition(to:with:)` to be called and update UI accordingly
 
+## Detecting if Device is Locked
+
+1. The first time your main app starts create a dummy file (let's call it ```ProtectionMonitor.dummy```) in a shared group container which is accessible for both the main app and the widget.
+	* You can create this file with ```NSFileManager.createFile(atPath:, contents:, attributes:)```
```

Comment:
Alternatively: ...create this file with `NSFileManager`'s `createFile(atPath:contents:attributes:)`. That at least prevents people from copying it directly.